packages/core/src/actors/index.ts

```diff
@@ -63,3 +69,9 @@ export function toActorRef<
     ...actorRefLike
   };
 }
+
+const nullBehavior = fromTransition((_) => undefined, undefined);
+
+export function createNullActor(): ActorRef<AnyEventObject, undefined> {
```

Comment:
q: why do we even need a null actor factory? can't we just export a null actor? 

```ts
export const nullActor = /* #__PURE__ */ interpret(nullBehavior);
```

Reply:
Equality checks. We don't want `someNullActor === anotherNullActor`

Reply:
Hm, I don't think that it matters too much - but perhaps to avoid weird issues with people trying to stop this actor etc it's best to create unique "instances".